scalableminds/webknossos
# Experience domain field in task creation as select component

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- the "change user experience"-modal should work as before
- includes a bug fix, that disables entering domains with less than 3 letters in the modal

- open the task creation
- without selecting a exp domain and submitting the form should show an typical FieldDecorator error
- selecting on should work and remove the waring from the FieldDecorator
- create a task
- check in the tasks view whether the exp domain is correct

### Issues:
- fixes #3031

------
- ~~~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~~~
- ~~~[ ] Updated [documentation](../blob/master/docs) if applicable~~~
- ~~~[ ] Needs datastore update after deployment~~~
